### ISSUE TITLE
ERROR in ./node_modules/@magenta/music/es6/index.js Module not found: Error: Can't resolve 'tf'

### ISSUE THREAD

**Issue by curiouscod3** [tensorflow/magenta-js#356]
I am using magenta-js for my Angular(ver. 8) project.
I am pretty sure that I've installed dependencies required.
I got this error. 
**"ERROR in node_modules/@magenta/music/es5/coconet/model.d.ts:21:77 - error TS2307: Cannot find module '../protobuf/proto'."**

When I inspect the node_modules, @magenta/music has es5 and es6 folders. 
I changed some in package.json from es5 to es6
`  "main": "es6/index.js",
  "module": "es6/index.js",
  "types": "es6/index.d.ts",
`

I've still got another Error.. 
I haven't seen this Error in the previous version. 
**`ERROR in ./node_modules/@magenta/music/es6/index.js
Module not found: Error: Can't resolve 'tf' `**

**Comment by curiouscod3** [comment]
I found a quick dirty solution for my Angular project; however, I don't recommend this .
**"add paths for the tf in tsconfig.json"**

The previous magenta-js version(1.15) only had es5 folder.
Current version has two different versions es5/es6; I think this is the main key of this issue to resolve this.

**Comment by notwaldorf** [comment]
Can you show me how you're importing this maybe? I haven't modified the es5 folder in any way, so if you were having problems before, you will still have problems now. If you're using the es6 folder, then you have to import the Tone/TF dependencies yourself, like in this example: https://github.com/tensorflow/magenta-js/blob/master/music/README.md#using-a-smaller-es6-bundle-for-just-the-code-you-need

**Comment by curiouscod3** [comment]
As I said, I am using Angular(v8) framework. This framework has own dependency mechanism unlike other frameworks or vanilla javascript. 

Actually, I used to work a magenta-js project with Angular(v7) + magenta.js(1.5.0 ver.).
There was no problem. As you know, in the previous version of magenta.js, there was only one folder(es5); there was no es6 folder. 

Since I am using Angular(ver. 8) + new magenta.js(^1.11.1), I have this issue.

**I installed all dependencies including TFJS/Tone.js. This is not the case that I didn't import Tone/TF dependencies.  Like I said, there was no issue like this in the previous version.** 
Anyway, if I put this in tsconfig.json, the error is gone!!
`"paths": {
      "tf" : [
        "node_modules/@tensorflow/tfjs"
      ]`

package.json
`{
  "name": "my-magenta-lab",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "browser": {
    "fs": false,
    "path": false,
    "os": false
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "~8.2.5",
    "@angular/cdk": "~8.2.0",
    "@angular/common": "~8.2.5",
    "@angular/compiler": "~8.2.5",
    "@angular/core": "~8.2.5",
    "@angular/flex-layout": "^8.0.0-beta.27",
    "@angular/forms": "~8.2.5",
    "@angular/material": "^8.2.0",
    "@angular/platform-browser": "~8.2.5",
    "@angular/platform-browser-dynamic": "~8.2.5",
    "@angular/router": "~8.2.5",
    "@magenta/music": "^1.11.1",
    "@tensorflow/tfjs": "^1.2.9",
    "buffer": "^5.4.3",
    "file-saver": "^2.0.2",
    "hammerjs": "^2.0.8",
    "lodash": "^4.17.15",
    "rxjs": "~6.4.0",
    "save": "^2.4.0",
    "tone": "^13.8.25",
    "tslib": "^1.10.0",
    "webmidi": "^2.5.1",
    "zone.js": "~0.9.1"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.803.4",
    "@angular/cli": "~8.3.4",
    "@angular/compiler-cli": "~8.2.5",
    "@angular/language-service": "~8.2.5",
    "@types/file-saver": "^2.0.1",
    "@types/jasmine": "~3.3.8",
    "@types/jasminewd2": "~2.0.3",
    "@types/node": "~8.9.4",
    "@types/webmidi": "^2.0.3",
    "codelyzer": "^5.0.0",
    "jasmine-core": "~3.4.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~4.1.0",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-jasmine": "~2.0.1",
    "karma-jasmine-html-reporter": "^1.4.0",
    "protractor": "~5.4.0",
    "ts-node": "~7.0.0",
    "tslint": "~5.15.0",
    "typescript": "~3.5.3"
  }
}
`

**Comment by notwaldorf** [comment]
I can add those paths to the tsconfig! Lemme check locally to make sure nothing bad happens if i do that

**Comment by curiouscod3** [comment]
@notwaldorf  I put the paths in my Angular's tsconfig.json, not in @magenta/music.

**Comment by notwaldorf** [comment]
Oh, my bad. Unfortunately I don't know anything about Angular, so I don't know how to fix it! Do you have any ideas?

**Comment by curiouscod3** [comment]
In the Angular side, I can fix like this. For the better way, @magenta/music package should provide itself automatically.  Why don't you put the paths in magenta.js  project?  You can give it a try.

**Comment by curiouscod3** [comment]
@notwaldorf 
Could you change the package.json like this?
Because I get this Error
**`es5/coconet/model.d.ts:21:77 - error TS2307: Cannot find module '../protobuf/proto'.`**

```javascript
**"main": "es6/index.js",**
  "module": "es6/index.js",
  **"types": "es6/index.d.ts",**
```

**Comment by notwaldorf** [comment]
I think this should be fixed by https://github.com/tensorflow/magenta-js/pull/365

**Comment by notwaldorf** [comment]
I released 1.11.3 which should have a fix for this. I also made an example repo to test it: https://github.com/notwaldorf/example-magenta-in-ts
